MiniApp/Classes/Models/MiniAppInfo.swift

```diff
@@ -0,0 +1,7 @@
+/// Mini App information
```

Comment:
Your models should be immutable i.e. non-optional.

Reply:
Done